tests/ui.test.ts

```diff
@@ -66,4 +79,51 @@ test.describe('FieldSelect', () => {
 		await expect(page.getByRole('option')).toHaveCount(0);
 		await expect(page.getByTitle('Clear')).not.toBeVisible();
 	});
+
+	test('models are correctly grouped and sorted', async ({ page }) => {
+		const newSessionButton = page.getByTestId('new-session');
+		await page.goto('/settings');
+		await mockTagsResponse(page);
```

Comment:
I had to do this because If I did `await page.goto('/sessions/qbhc0q');` instead of going to `/settings` first, the `Run` button would never get enabled.

Reply:
For some reason the `Run` button is still disabled when the tests are run in the GH Action. Debugging...

Reply:
Gotcha, yeah, let's get that sorted out before merging.

Reply:
All tests pass if I set `retries: 2` in `playwright.config.ts` 🤔